lib/inspec/config.rb

```diff
@@ -344,6 +350,8 @@ def merge_options
       # Highest precedence: merge in any options defined via the CLI
       options.merge!(@cli_opts)
 
+      options.merge!(@cfg_file_contents)
```

Comment:
This seems off topic for this commit, no? Is it actually related?

Reply:
This all comes down to what is available to input plugins.  When I was debugging, and trying to figure out exactly how this all works :), The final_options was all that was available.  I would take some KT on this if there is another way.   The user scenario was to have one configuration file for Inspec to maintain.

Reply:
As the first public contributor known to be writing an input plugin, I'd love to hear more about your experiences and how we can make it a convenient API. It is extremely minimal at the moment.

Reply:
As of yet, I have not done much with it.   As I am taking a MVP iterative approach to some use cases, I will be building out a better story.  Basically, our silo'ed ecosystem does not align well with a centralized auditor.  Too many heads.  What I am wishing to build out is a 'Oracle' service that Inspec can chat with to get quorum agreed upon values.  As of this time, the only down side was that I had no method to configure the plugin without statically defining a file.